sdk/openai/openai/README.md

```diff
@@ -34,6 +34,8 @@ Key links:
 - [Product documentation](https://learn.microsoft.com/azure/cognitive-services/openai)
 - [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/openai/openai/samples/v1-beta)
 
+Note: The latest release version of @azure/openai at 1.0.0-beta.12 upgraded the default API version to `2024-03-01-preview`. This has many breaking changes for Bring Your Own Data features. Please refer to the [changelog][changelog] for a list of the breaking changes and follow the [Azure OpenAI sample][byod_sample] to use your own data.
```

Comment:
Do we need to have this in the readme?

Reply:
I'd like to add it somewhere in the Readme so that it is more visible to the customers.